lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/FastqsToBam.pm

```diff
@@ -35,7 +36,7 @@ class Genome::InstrumentData::Command::Import::WorkFlow::FastqsToBam {
         output_bam_path => {
             is => 'Text',
             calculate_from => [qw/ working_directory sample_name /],
-            calculate => q( return $working_directory.'/'.$sample_name.'.bam'; ),
+            calculate => q| return File::Spec->join($working_directory, Genome::Utility::Text::sanitize_string_for_filesystem($sample_name).'.bam'); |,
```

Comment:
This module should `use Genome::Utility::Text` in order to prevent a `Use of inherited AUTOLOAD for non-method ... is deprecated` warning (or, moreover, to not potentially rely on a deprecated feature :smile:).